pkg/scheduler/nodeinfo/snapshot/snapshot.go

```diff
@@ -189,7 +189,7 @@ func (n *nodeInfoLister) HavePodsWithAffinityList() ([]*schedulernodeinfo.NodeIn
 
 // Returns the NodeInfo of the given node name.
 func (n *nodeInfoLister) Get(nodeName string) (*schedulernodeinfo.NodeInfo, error) {
-	if v, ok := n.snapshot.NodeInfoMap[nodeName]; ok && v.Node() != nil {
+	if v, ok := n.snapshot.NodeInfoMap[nodeName]; ok {
```

Comment:
Uhm... actually I should keep this, as an AddPod could arrive its corresponding AddNode.